cluster/ec2_pke.go

```diff
@@ -874,31 +882,30 @@ func (c *EC2ClusterPKE) GetBootstrapCommand(nodePoolName, url string, urlInsecur
 	}
 	switch cloudProvider {
 	case string(internalPke.CNPAmazon):
-		// match subnet
-		if len(subnets) > 0 {
-			s, err := c.GetAWSClient()
-			if err != nil {
-				return "", err
-			}
-			c := ec2.New(s)
-
+		subnetId := ""
```

Comment:
This block would be worth a few lines of comment. The mists of time may blur the intent behind this logic.